internal/Picker/PickerItem.js

```diff
@@ -0,0 +1,34 @@
+import kind from '@enact/core/kind';
+import React from 'react';
+
+import Marquee from '@enact/ui/Marquee';
+
+import css from './Picker.module.less';
+
+/**
+ * Renders an Agate-styled Picker Item without any behavior.
+ *
+ * @class PickerItemBase
+ * @memberof agate/PickerItem
+ * @ui
+ * @public
```

Comment:
`PickerItemBase` is an internal component. So it might be better to define it as private.